spec/reek/smells/attribute_spec.rb

```diff
@@ -17,6 +17,47 @@
       ctx = Reek::Core::CodeContext.new(nil, Reek::Source::SourceCode.from(src).syntax_tree)
       expect(@detector.examine_context(ctx)).to be_empty
     end
+
+    shared_examples_for 'no attribute found' do
+      before :each do
+        ctx = Reek::Core::CodeContext.new(nil, Reek::Source::SourceCode.from(@src).syntax_tree)
+        @smells = @detector.examine_context(ctx)
+      end
+
+      it 'records no attribute' do
+        expect(@smells.length).to eq(0)
```

Comment:
OTOH, since this was how the whole spec file was before, we can also leave this change for later.